### PR TITLE
[FEAT/NOTIFICATION]: Create Notification Page

### PR DESCRIPTION
**Pull request by baohuy2k3**
## Description

- Init Notification

## Project

- [x] Website
- [ ] API
- [ ] Documentation
- [ ] Other... (please describe)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other... (please describe)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B
- [x] No tests needed

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Other... (please describe)

### CODE REVIEW

File: apps/website/src/features/Notification/index.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  const breadcrumbs = [
+    <Link key="1" color="inherit" to="/">
+      Trang chủ
+    </Link>,
+    <Typography key="2" color="text.primary">
+      Giới thiệu
+    </Typography>,
+  ];
+  return (
+    <div>
```
Comment (qoucname2202) [inline apps/website/src/features/Notification/index.tsx:24]: @baohuy2k3 Using react fragment element replace for the div element

File: apps/website/src/@types/images.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import dvc_banner from '../assets/images/notification/dvc_banner.jpg';
+import dvc from '../assets/images/notification/dvc.png';
+import thatnghiep from '../assets/images/notification/GQ-Huong-tro cap-that nghiep.png';
+import loidangky from '../assets/images/notification/Loi dang ky tk.png';
+import lptb from '../assets/images/notification/lptb.png';
+
+type bannerNotificationType = {
+    DVCBanner: string,
```
Comment (qoucname2202) [inline apps/website/src/@types/images.ts:8]: @baohuy2k3 Convert all Vietnamese variables to English variables

File: apps/website/src/@types/images.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    DVC: string,
+    Thatnghiep: string,
+    Loidangky: string,
+    LPTB: string,
+}
+
+export const bannerNotification: bannerNotificationType = {
+    DVCBanner: dvc_banner,
```
Comment (qoucname2202) [inline apps/website/src/@types/images.ts:16]: @baohuy2k3 Convert all Vietnamese variables to English variables

File: apps/website/src/features/Notification/index.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import { Container } from '@mui/system';
+import useMetadata from '../../hooks/useMetadata';
+import { Link } from 'react-router-dom';
+import { Breadcrumbs, Grid, Typography } from '@mui/material';
+import { FaAngleRight } from 'react-icons/fa';
+import ItemsNotification from './components/ItemsNotification';
+import trongdong from '../../assets/images/notification/trongdong.png';
```
Comment (qoucname2202) [inline apps/website/src/features/Notification/index.tsx:7]: @baohuy2k3 Give the variable an english name containing the path to the directory

File: apps/website/src/@types/images.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import dvc_banner from '../assets/images/notification/dvc_banner.jpg';
+import dvc from '../assets/images/notification/dvc.png';
+import thatnghiep from '../assets/images/notification/GQ-Huong-tro cap-that nghiep.png';
+import loidangky from '../assets/images/notification/Loi dang ky tk.png';
```
Comment (qoucname2202) [inline apps/website/src/@types/images.ts]: @baohuy2k3 Image names cannot contain white spaces

**Review by qoucname2202 — CHANGES_REQUESTED** [review]
@baohuy2k3 Fix code style for each file has comment

**Review by foxminchan — APPROVED** [review]

**Review by qoucname2202 — APPROVED** [review]
@foxminchan approved merger

**Pull request merged**